types/stripe/index.d.ts

```diff
@@ -4073,12 +4073,12 @@ declare namespace Stripe {
 
     namespace bankAccounts {
         interface IBankAccount extends IBankAccountHash {
-            account: string;
+            account?: string;
```

Comment:
Stripe does not always return this value. There is no documentation on what this value is at https://stripe.com/docs/api/customer_bank_accounts/object#customer_bank_account_object-account